src/dialects/mssql/index.js

```diff
@@ -138,6 +140,18 @@ assign(Client_MSSQL.prototype, {
     })
   },
 
+  // sets a request input parameter. Detects bigints and sets type appropriately.
+  _setReqInput(req, i, binding) {
+    if (typeof binding == 'number' && (binding < -2147483648 || binding > 2147483647)) {
+      if (binding < -9007199254740991 || binding > 9007199254740991) {
```

Comment:
Would be nice for the code to give names for the magic numbers used e.g. MIN/MAX_INT4 MIN/MAX_SAFE_INTEGER. Anyways we are using babel to compile sources to ES5 so I suppose Number.MAX_SAFE_INTEGER / Number.isSafeInteger() should work too.